src/color.js

```diff
@@ -286,21 +286,35 @@ exports.byElement = function() {
   }, null, null);
 };
 
-exports.bySS = function() {
+exports.bySS = function(grad) {
+  var palette;
+  if (grad) {
+    palette = {
+      C: grad._colors[0],
+      H: grad._colors[1],
+      E: grad._colors[2],
```

Comment:
Rather than providing the colors as an array, I think it would make sense to pass them as a dictionary. This is way easier to read.
```js
color.bySS([one, two, three])
```

would then become

```js
color.bySS({ H : one, C : two, E : three})
```